internal/handler/commands.go

```diff
@@ -77,6 +77,10 @@ func (h *Handler) initCommands() {
 			Handler: h.MsgCreateIndexes,
 			Help:    "Creates indexes on a collection.",
 		},
+		"createUser": {
+			Handler: h.MsgCreateUser,
+			Help:    "Creates a new user on the database where you run the command.",
+		},
```

Comment:
Those commands are listed down below inside of `EnablenewAuth` if, so I assume they should be only accessible when running FerretDB with `-test-enable-new-auth`, i can be wrong though

Reply:
You are correct indeed!